protocol/v2/types/crypto.go

```diff
@@ -7,14 +7,32 @@ import (
 	spectypes "github.com/bloxapp/ssv-spec/types"
 	"github.com/herumi/bls-eth-go-binary/bls"
 	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"go.uber.org/zap"
 )
 
+var (
+	metricsSignaturesVerifications = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "ssv_signature_verifications",
+		Help: "Number of signatures verifications",
+	}, []string{})
+)
+
+func init() {
+	logger := zap.L()
+	if err := prometheus.Register(metricsSignaturesVerifications); err != nil {
+		logger.Debug("could not register prometheus collector")
+	}
+}
+
 // VerifyByOperators verifies signature by the provided operators
 // This is a copy of a function with the same name from the spec, except for it's use of
 // DeserializeBLSPublicKey function and bounded.CGO
 //
 // TODO: rethink this function and consider moving/refactoring it.
 func VerifyByOperators(s spectypes.Signature, data spectypes.MessageSignature, domain spectypes.DomainType, sigType spectypes.SignatureType, operators []*spectypes.Operator) error {
+	metricsSignaturesVerifications.WithLabelValues().Inc()
```

Comment:
good metric, though is this the only place we verify sigs?

Reply:
@GalRogozinski do we need this to track other signature verification methods as well?

Reply:
We also track RSA sigs in another metric

Are there more places where we do BLS verifications?
This looks like the main function to me

Reply:
well I guess better to add